stdlib/public/stubs/LibcShims.cpp

```diff
@@ -55,10 +71,14 @@ int _swift_stdlib_close(int fd) { return close(fd); }
 #if defined(__APPLE__)
 #include <malloc/malloc.h>
 size_t _swift_stdlib_malloc_size(const void *ptr) { return malloc_size(ptr); }
-#elif defined(__GNU_LIBRARY__) || defined(__CYGWIN__)
+#elif defined(__GNU_LIBRARY__) || defined(__CYGWIN__) || defined(_MSC_VER)
 #include <malloc.h>
 size_t _swift_stdlib_malloc_size(const void *ptr) {
+#if defined(_MSC_VER)
```

Comment:
Please do another `#elif` on the top level in the existing chain instead of splitting the function definition with `#if`.